cnd/src/db/integration_tests.rs

```diff
@@ -54,17 +54,17 @@ macro_rules! db_roundtrip_test {
                         db.save(saved_accept.clone()).await?;
 
                         let loaded_swap = Retrieve::get(&db, &swap_id).await?;
-                        let (loaded_request, loaded_accept) = db.load_accepted_swap(swap_id).await?;
+                        let (loaded_request, loaded_accept, _at) = db.load_accepted_swap(&swap_id).await?;
```

Comment:
📣  Why not testing the timestamp too?

Reply:
I could not think of any meaningful way to do so, that's why I added `accepted_swap_has_valid_timestamp()` in `cnd/tests`.  Open to suggestions.